src/bin/init/src/main.rs

```diff
@@ -242,6 +242,17 @@ fn exec2(name: &str, id: ObjID) -> Option<ObjID> {
     //println!("ELF: {:?}", elf);
 }
 
+fn exec_n(name: &str, id: ObjID, args: &[&str]) {
```

Comment:
This name is confusing to me.  Can we expand the n or make a block comment?

Reply:
I'm planning to totally overhaul this in another patch, didn't want to mix in all the changes here, this was the path of least resistance for this PR